src/indigo/less/inline-edit-text-input.less

```diff
@@ -0,0 +1,77 @@
+.inline-edit-text-input {
+  display: inline;
+  position: relative;
+
+  &-form {
+    display: inline;
+  }
+
+  &-buttons {
+    position: absolute;
+    right: 10px;
+    top: -3px;
+
+    .fa-spin {
+      position: absolute;
+      top: 5px;
+      right: -35px;
+      background: #fff;
+    }
+  }
+
+  &-cancel {
+    padding: 0 5px 0 5px;
+    font-size: 120%;
+    color: #e1e1e9;
+
+    &:hover,
+    &:focus,
+    &:active {
+      text-decoration: none;
+      color: darken(#e1e1e9, 20%);
+    }
+
+    .kbc-icon-cross {
+      position: relative;
+      top: 2px;
+    }
+  }
+
+  &-submit {
+    padding: 5px 10px;
+    font-size: 60%;
+    border-radius: 15px;
+    [class^="kbc-icon-"] {
+      font-size: 200%;
+    }
+  }
+
+  &-link {
```

Comment:
Tady chvalim ze jsi pouzil lexikalni spojovani classnames.
Vim ze tohle neni ultimatni styl psani css - resp ma to svoje limity.
Pokud to udrzime v 1 (max 2) urovne zanoreni tak to bude dobry reseni.